charts/topolvm/templates/scheduler/poddisruptionbudget.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.scheduler.podDisruptionBudget.enabled }}
+{{ if .Values.scheduler.enabled }}
+  {{ if .Values.scheduler.podDisruptionBudget.enabled }}
```

Comment:
I recommend using and function.
https://helm.sh/docs/chart_template_guide/function_list/#and

Reply:
@toshipp 
Other files in templates/scheduler aren't used `and` function. I mimicked those files after thinking about it a bit. Would I rewrite all of the files using `and` function?

Reply:
Oh, I missed it. Other templates except for scheduler use `and` function rather than nesting `if`, I think it is better. So let's rewrite templates to use `and`